og_ui/og_ui.services.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  access_check.og.user_access_group:
```

Comment:
Ha, fair enough. I think the move could make sense. Let's do that in a follow up as it's a fundamental change to OG.

@amitaibu I didn't think someone could get any wiser but you managed it ;)